engine/apps/alerts/incident_appearance/renderers/slack_renderer.py

```diff
@@ -169,7 +169,7 @@ def _get_buttons_attachments(self):
     def _get_buttons_blocks(self):
         AlertGroup = apps.get_model("alerts", "AlertGroup")
         buttons = []
-        if self.alert_group.maintenance_uuid is None:
+        if not self.alert_group.is_maintenance_incident:
```

Comment:
Consider renaming `is_maintenance_incident` to `is_maintenance_alert_group` for consistency